index.html

```diff
@@ -58,5 +58,179 @@ <h1>I'm Collins. Glad to see you!</h1>
         <li><i class="fa-brands fa-facebook-f"></i></li>
       </ul>
     </section>
+    <section class="card">
+      <div class="grid-items">
+        <div class="cards">
+          <img src="./Images/Snapshoot-Portfolio.png" alt="" />
+        </div>
+        <div class="cards">
+          <h1>Tonic</h1>
+          <div class="cards-details">
+            <div class="project">
+              <span>Canopy</span>
+              <ul class="listA">
+                <li class="listA-items">BackEnd</li>
+                <li class="listA-items">2015</li>
+              </ul>
+            </div>
+
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
```

Comment:
This text looks a bit bolder than it should be please do your best to make your design looks exactly like the template